index.bs

```diff
@@ -493,15 +493,21 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dd>This reflects the supported range of <a>contrast</a>. Values are numeric.  Increasing values indicate increasing contrast.</dd>
 
   <dt><dfn dict-member for="MediaTrackCapabilities"><code>pan</code></dfn></dt>
-  <dd>This reflects the <a>pan</a> value range supported by the UA and by the track.
+  <dd>
+    This reflects the <a>pan</a> value range supported by the UA and by the track.
 
-  If the track has been created without <a>requesting permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true or if that permission request is denied, the track does not support <a>pan</a>.
-  In that case the UA MUST NOT expose the <a>pan</a> value range.
+    If the track has been created without <a>requesting permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true or if that permission request is denied, the track does not support <a>pan</a>.
+    In that case the UA MUST NOT expose the <a>pan</a> value range but MAY provide an empty {{MediaSettingsRange}} dictionary to indicate that the underlying video source supports <a>pan</a>.
 
-  <div class="note">
-    It does not matter whether the algorithm to <a>request permission to use</a> a PermissionDescriptor returns the current permission state without user interaction (when the current state is not "prompt") or asks the user's permission.
-    In both cases, the permission to use is requested and either granted or denied.
-  </div></dd>
+    <div class="note">
+      It does not matter whether the algorithm to <a>request permission to use</a> a PermissionDescriptor returns the current permission state without user interaction (when the current state is not "prompt") or asks the user's permission.
+      In both cases, the permission to use is requested and either granted or denied.
+    </div>
+
+    <div class="note">
+      Even if the UA provides an empty {{MediaSettingsRange}} dictionary to indicate that the underlying video source supports <a>pan</a>, that support can be utilized only after a new {{getUserMedia()}} call which resolves a {{MediaStream}} object which contains a video track which supports <a>pan</a>.
```

Comment:
Shall we have this note applied to `tilt` and `zoom` as well?

Reply:
Done.